xdsl/interactive/app.py

```diff
@@ -143,6 +154,12 @@ def on_mount(self) -> None:
         for n, _ in ALL_PASSES:
             self.passes_list_view.append(ListItem(Label(n), name=n))
 
+        # initialize GUI with an interesting input IR and pass application
+        self.input_text_area.load_text(self.INITIAL_IR_TEXT)
+        self.pass_pipeline = tuple(
+            (*self.pass_pipeline, convert_func_to_riscv_func.ConvertFuncToRiscvFuncPass)
```

Comment:
I would recommend going with something that will be relevant to any project rather than just the riscv thing, it seems like `dce` or `canonicalize` might be more universally applicable. I also think there's a case to be made for no passes being interesting enough, just to see how it parses and re-prints the code. There's also fewer actions to make to remove the preloaded pass in that case.